### PR TITLE
Update KubevirtHcoSingleStackIPv6

### PR DESCRIPTION
**Pull request by apinnick**
Updating runbook because I think the command might not be correct. See [Viewing the cluster network configuration](https://docs.openshift.com/container-platform/4.13/networking/cluster-network-operator.html#nw-cno-view_cluster-network-operator).

### CODE REVIEW

**Comment by apinnick** [comment]
@phoracek Can you check this?

File: alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md [hunk 1]
```diff
@@ -11,13 +11,25 @@ You cannot create virtual machines.
 
 ## Diagnosis
 
-- Obtain the network details for the cluster by running the following command:
+- Check the cluster network by running the following command:
```
Comment (phoracek) [inline alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md:14]:
```suggestion
- Check the cluster network configuration by running the following command:
```

File: alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md [hunk 2]
```diff
@@ -11,13 +11,25 @@ You cannot create virtual machines.
...
   ```shell
-  $ oc describe network cluster
+  $ oc describe network.config/cluster
```
Comment (phoracek) [inline alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md:16]:
```suggestion
  $ oc get network.config cluster -o yaml
```

The format of the output bellow is YAML, so `oc get` must be used to get it

File: alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md [hunk 3]
```diff
@@ -11,13 +11,25 @@ You cannot create virtual machines.
...
+  Example output:
+  ```text
+  apiVersion: config.openshift.io/v1
+  kind: Network
+  metadata:
+    name: cluster
+  spec:
+    clusterNetwork:
+      - cidr: fd02::/48
+        hostPrefix: 64
```
Comment (phoracek) [inline alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md:29]:
```suggestion
    - cidr: fd02::/48
      hostPrefix: 64
```

**Review by phoracek — CHANGES_REQUESTED** [review]

File: alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md (hunk 2 above)
Comment (phoracek) [reply alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md:16]: `oc get network cluster -o yaml` works for me too. But this way we at least won't get it mixed up with `operator.openshift.io/v1`

**Comment by apinnick** [comment]
@sradco 
I updated the PR with Petr's changes. PR is ready for your review.

**Pull request merged**